developer-docs-site/docs/reference/telemetry.md

```diff
@@ -3,32 +3,101 @@ title: "Telemetry"
+- `IP_ADDRESS`
+- `TELEMETRY_TOKEN`
+- **Total event keys**: 8
+- **Frequency**: Every minute.
+
+### Build information
+
+Rust build information including the versions of Rust, cargo, build target architecture and the build tag.
+
+- **Build information keys (15**): [https://github.com/aptos-labs/aptos-core/blob/main/crates/aptos-telemetry/src/build_information.rs#L12-L26](https://github.com/aptos-labs/aptos-core/blob/main/crates/aptos-telemetry/src/build_information.rs#L12-L26).
+- `IP_ADDRESS`
+- `TELEMETRY_TOKEN`
+- **Total event keys**: 17
+- **Frequency**: Every 1 hour.
+
+### System information
+
+System information including operating system information (including versions), hardware information and resource utilization (including CPU, memory and disk).
+
+- **System information keys (19)**: [https://github.com/aptos-labs/aptos-core/blob/main/crates/aptos-telemetry/src/system_information.rs#L14-L32](https://github.com/aptos-labs/aptos-core/blob/main/crates/aptos-telemetry/src/system_information.rs#L14-L32)
+- `IP_ADDRESS`
+- `TELEMETRY_TOKEN`
+- **Total event keys**: 21
+- **Frequency**: Every 5 minutes.
+
+
+
+## Disabling telemetry
+
+On macOS and Linux, you can set the following environment variables to control the metrics sent by your node. For example, to disable all telemetry, set the  `APTOS_DISABLE_TELEMETRY` environment variable to `true` as shown below:
+
+```bash
 export APTOS_DISABLE_TELEMETRY=true
 ```
 
-The above example only disables telemetry for a single session or terminal. To disable it everywhere, you must do so at shell startup.
+The above example only disables telemetry for a single session in the current terminal where you ran the above command. To disable it permanently on your node, include it in your startup profile, as below: 
 
-```
+```bash
 echo "export APTOS_DISABLE_TELEMETRY=true" >> ~/.profile
 source ~/.profile
 ```
 
-# Types of information collected
+:::tip All telemetry is ON by default.
+All the below variables are set by default to `false`, i.e., sending of these telemetry metrics is enabled. Set them to `true` to disable telemetry.
+:::
+
+- `APTOS_DISABLE_TELEMETRY`: This disables all telemetry emission from the node including sending to the GA4 service.
+- `APTOS_FORCE_ENABLE_TELEMETRY`: This overrides the chain ID check and forces the node to send telemetry regardless of whether remote service accepts or not.
+- `APTOS_DISABLE_TELEMETRY_PUSH_METRICS`: This disables sending the [Prometheus](https://prometheus.io/) metrics.
+- `APTOS_DISABLE_TELEMETRY_PUSH_LOGS`: This disables sending the logs.
+- `APTOS_DISBALE_TELEMETRY_PUSH_EVENTS`: This disables sending the custom events to [BigQuery](https://cloud.google.com/bigquery).
+- `APTOS_DISABLE_LOG_ENV_POLLING`: This disables the node local task that polls the service for `RUST_LOG_TELEMETRY` config.
```

Comment:
Suggest wording this as "This disables the dynamic ability to send verbose logs"